R/btw_client.R

```diff
@@ -136,7 +136,7 @@ btw_client <- function(..., client = NULL, tools = NULL, path_btw = NULL) {
 
   if (!skip_tools) {
     maybe_quiet <- if (is.null(tools)) suppressMessages else I
-    maybe_quiet(btw_register_tools(client, tools = config$tools))
+    maybe_quiet(client$set_tools(tools = btw_tools(config$tools)))
```

Comment:
I don't think we need this `maybe_quiet()` anymore, it was about suppressing the "Registered tool" cli messages that we aren't emitting anymore.